vsintegration/src/FSharp.Editor/Options/EditorOptions.fs

```diff
@@ -194,3 +188,56 @@ module internal OptionsUI =
         inherit AbstractOptionPage<FormattingOptions>()
         override _.CreateView() =
             upcast FormattingOptionsControl()
+
+[<AutoOpen>]
+module EditorOptionsExtensions =
+
+    type Project with
+
+        member this.IsFSharpCrossProjectReferencingEnabled =
```

Comment:
I think the naming should match the setting i..e. `AreInMemoryCrossProjectReferencesEnabled``

Reply:
Makes sense to me.